examples/textures/textures_bunnymark.ml

```diff
@@ -25,7 +25,7 @@ let rec loop bunnies bunnies_count tex_bunny =
   | false ->
       let open Raylib in
       let bunnies, bunnies_count =
-        if is_mouse_button_down MouseButton.Left then
+        if is_mouse_button_down MouseButton.Left_button then
```

Comment:
Similar to the `Key` issue, how do you feel about leaving the `_button`? Imo `MouseButton.Left` is more intuitive.

Reply:
Sounds good.